### PR TITLE
add user context and navbar

### PR DESCRIPTION
**Pull request by Tharun0102**

### CODE REVIEW

File: pages/people.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-    <>
-      <div className="bg-main-bg  text-white min-h-screen min-w-full">
-        <h1 className="">People</h1>
-      </div>
-    </>
-  );
+	return (
+		<>
+			<div className="bg-main-bg  text-white min-h-screen min-w-full">
+				<h1 className="">People</h1>
```
Comment (lakshya-dhariwal) [inline pages/people.tsx:7]: Is your prettier at default settings. if not do that, otherwise we will get unwanted changes in every commit

**Comment by lakshya-dhariwal** [comment]
I will take over the responsive nav feature after solving the prettier issue

**Comment by Tharun0102** [comment]
> I will take over the responsive nav feature after solving the prettier issue

Okay

**Review by lakshya-dhariwal — APPROVED** [review]

**Comment by lakshya-dhariwal** [comment]
Looks good to me 🚀

**Pull request merged**